README.md

```diff
@@ -1,16 +1,10 @@
 # Workflow run cleanup action
 
-This action cleans up previously running instances of a workflow
-on the same branch. This accomplishes the task of automatically
-cancelling CI runs on pushes to the same branch, which is a common
-feature in most CI systems but currently not possible with
-GitHub actions.
+This action cleans up previously running instances of a workflow on the same branch. This accomplishes the task of automatically cancelling CI runs on pushes to the same branch, which is a common feature in most CI systems but currently not possible with GitHub actions.
 
 ## Configuration
 
-None. The action uses the GitHub action environment variables
-to obtain the workflow name and branch. You must, however,
-set the `GITHUB_TOKEN` environment variable:
+None. The action uses the GitHub action environment variables to obtain the workflow name and branch. You must, however, create an [access token](https://github.com/settings/tokens) that includes the scope `repo`. Then, you can set the `GITHUB_TOKEN` environment variable:
```

Comment:
You don't strictly need to create a new access token - one is already provided for you automatically in github actions (see [here](https://help.github.com/en/actions/configuring-and-managing-workflows/authenticating-with-the-github_token#about-the-github_token-secret)). Maybe it would be better to word this differently, so it doesn't sound like you *have* to create a personal access token?